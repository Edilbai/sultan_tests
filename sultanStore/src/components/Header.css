
.address {
	height: 34px;
}
.header_top_right{
	max-width: 512px;
	
}
p{
	margin-left: 50px;
}
.icon{
	margin-right: 16px;
	margin-left: 40px;
}
.header_top{
	border-bottom: 1px solid rgba(63, 78, 101, 1);;
	padding: 12px;
}


@media(max-width:320px){

.address{
	display: none;
}
}
