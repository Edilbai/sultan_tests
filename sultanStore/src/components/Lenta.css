.lenta{
	margin-left: 100px;
	margin-top: 20px;
	color:  rgba(63, 78, 101, 1);;
}