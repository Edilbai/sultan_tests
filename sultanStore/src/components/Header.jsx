import React from 'react'

import '../main.css'
import '../components/Header.css'

const Header = () => {
	return (
		<div className='header'>
			<div className='d-flex header_top space-b address'>
				<div className='d-flex space-b'>
					<div className='d-flex space-a al-center'>
						<img
							className='icon'
							width={13}
							height={16}
							src='./img/icons/first.png'
						/>
						<div className='d-flex address column'>
							<b>г. Кокчетав, ул. Ж. Ташенова 129Б </b>
							<span>(Рынок Восточный)</span>
						</div>
					</div>
					<div className=' d-flex space-a al-center'>
						<img
							className='icon'
							width={16}
							height={13}
							src='./img/icons/second.png'
							alt=''
						/>
						<div className='address d-flex column	'>
							<b>opt.sultan@mail.ru</b>
							<span> На связи в любое время</span>
						</div>
					</div>
				</div>

				<div className='header_top_right al-center d-flex space-b '>
					<p>О компании</p>
					<p>Доставка и оплата</p>
					<p>Возврат</p>
					<p>Контакты</p>
				</div>
			</div>
		</div>
	)
}

export default Header
