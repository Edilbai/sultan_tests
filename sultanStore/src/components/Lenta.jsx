import React from 'react'
import { Link, Route, Routes } from 'react-router-dom'
import Corzina from '../pages/Corzina'
import Katalog from '../pages/Katalog'
import './Lenta.css'
import { useState } from 'react'

const Lenta = props => {
	return (
		<div>
			<div className='lenta'>
				<Link data-testid='main' to='/'>
					Главная
				</Link>
				<span> / </span>
				<Link data-testid='rabbish' to='/corzina'>
					Корзина
				</Link>
			</div>
		</div>
	)
}

export default Lenta
