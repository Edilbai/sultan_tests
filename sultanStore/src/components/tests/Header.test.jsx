import { screen, render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Header from '../Header'
import Lenta from '../Lenta'

describe('render Corzina', () => {
	it('to be text', () => {
		render(<Header />)
		const el = screen.getByText(/(Рынок Восточный)/i)
		expect(el).toBeInTheDocument()
	})

	it('Header snapshot', () => {
		const snap = render(<Header />)
		expect(snap).toMatchSnapshot()
	})
})
