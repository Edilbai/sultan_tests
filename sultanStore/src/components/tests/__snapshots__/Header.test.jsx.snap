// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Corzina Header snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="header"
      >
        <div
          class="d-flex header_top space-b address"
        >
          <div
            class="d-flex space-b"
          >
            <div
              class="d-flex space-a al-center"
            >
              <img
                class="icon"
                height="16"
                src="./img/icons/first.png"
                width="13"
              />
              <div
                class="d-flex address column"
              >
                <b>
                  г. Кокчетав, ул. Ж. Ташенова 129Б 
                </b>
                <span>
                  (Рынок Восточный)
                </span>
              </div>
            </div>
            <div
              class=" d-flex space-a al-center"
            >
              <img
                alt=""
                class="icon"
                height="13"
                src="./img/icons/second.png"
                width="16"
              />
              <div
                class="address d-flex column	"
              >
                <b>
                  opt.sultan@mail.ru
                </b>
                <span>
                   На связи в любое время
                </span>
              </div>
            </div>
          </div>
          <div
            class="header_top_right al-center d-flex space-b "
          >
            <p>
              О компании
            </p>
            <p>
              Доставка и оплата
            </p>
            <p>
              Возврат
            </p>
            <p>
              Контакты
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="header"
    >
      <div
        class="d-flex header_top space-b address"
      >
        <div
          class="d-flex space-b"
        >
          <div
            class="d-flex space-a al-center"
          >
            <img
              class="icon"
              height="16"
              src="./img/icons/first.png"
              width="13"
            />
            <div
              class="d-flex address column"
            >
              <b>
                г. Кокчетав, ул. Ж. Ташенова 129Б 
              </b>
              <span>
                (Рынок Восточный)
              </span>
            </div>
          </div>
          <div
            class=" d-flex space-a al-center"
          >
            <img
              alt=""
              class="icon"
              height="13"
              src="./img/icons/second.png"
              width="16"
            />
            <div
              class="address d-flex column	"
            >
              <b>
                opt.sultan@mail.ru
              </b>
              <span>
                 На связи в любое время
              </span>
            </div>
          </div>
        </div>
        <div
          class="header_top_right al-center d-flex space-b "
        >
          <p>
            О компании
          </p>
          <p>
            Доставка и оплата
          </p>
          <p>
            Возврат
          </p>
          <p>
            Контакты
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
